cli/lsp/testing/lsp_custom.rs

```diff
@@ -21,8 +21,9 @@ pub struct TestData {
   pub id: String,
   /// The human readable test to display for the test.
   pub label: String,
-  #[serde(skip_serializing_if = "Option::is_none")]
-  pub steps: Option<Vec<TestData>>,
+  #[serde(skip_serializing_if = "Vec::is_empty")]
+  #[serde(default)]
+  pub steps: Vec<TestData>,
```

Comment:
This changes the deserialization of this structure.  I can't remember off hand if this is ever sent from the client, but why arbitrarily break the contract with the client?